packages/ombits-client/src/components/header.tsx

```diff
@@ -0,0 +1,96 @@
+import React, { useState, useCallback } from 'react';
+import { useCurrentRoute } from 'react-navi';
+
+import { Nav } from './nav';
+
+export const Header = () => {
+  const route = useCurrentRoute();
+  const { navTabText } = route.state;
```

Comment:
@remotealex we are maintaining this navTabText state ? and which TabText it's pointing to ?

Reply:
state comes from the routes, check it out here:

https://github.com/remotealex/ombits/pull/14/files/713ee7b29c12e1937be6fa8ecf81abc3e12eea7d#diff-223c6d316415e685dee9b06b2a14e6acR10